### PR TITLE
feat(compute): simplify Zonal MIG example to match gcloud example

### PR DESCRIPTION
**Pull request by askubis**
## Description

This is sample is for matching the following gcloud command in https://cloud.google.com/compute/docs/instance-groups/create-zonal-mig

```
gcloud compute instance-groups managed create example-group \
    --base-instance-name test \
    --size 3 \
    --template an-instance-template \
    --zone us-central1-f
```

Fixes #<ISSUE-NUMBER>

Note: If you are not associated with Google, open an issue for discussion before submitting a pull request.

## Checklist

**Readiness**

- [X] Yes, **merge** this PR after it is approved
- [ ] No, don't **merge** this PR after it is approved

**Style**

- [X] My sample follows the rules described for Terraform in the [Effective Samples style guide](https://googlecloudplatform.github.io/samples-style-guide/)
- [ ] My sample follows the other requirements and best practices in the [Contributing
guide](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#other-requirements-and-best-practices)

**Testing**

- [X] I have performed tests described in the [Contributing guide](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md):

   - [X] **[Tests](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#set-up-the-test-environment)** pass: `terraform apply`
   - [X] **[Lint](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#linting-and-formatting)** pass: `terraform fmt` check

**Intended location**

- [X] Yes, this sample will be (or already is) included on cloud.google.com
- [ ] No, this sample won't be included on cloud.google.com
      Reason:

**API enablement**

- [X] If the sample needs an API enabled to pass testing, I have added the service to the [Test setup file](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/test/setup/main.tf)

**Review**

- [X] If this sample adds a new directory, I have added codeowners to the [CODEOWNERS file](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/.github/CODEOWNERS)

### CODE REVIEW

File: compute/zonal_instance_group_manager/main.tf [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2022 Google LLC
+ * Copyright 2023 Google LLC
```
Comment (gericdong) [inline compute/zonal_instance_group_manager/main.tf:2]: We don't typically need to update the license year unless it's a correction.

**Review by gericdong — COMMENTED** [review]
Please investigate the test errors. Thanks.

File: compute/zonal_instance_group_manager/main.tf (hunk 1 above)
Comment (askubis) [reply compute/zonal_instance_group_manager/main.tf:2]: it is a correction, there was even a comment in my previous PR, but it got submitted anyway. 
https://github.com/terraform-google-modules/terraform-docs-samples/pull/485

File: compute/zonal_instance_group_manager/main.tf [hunk 2]
```diff
@@ -25,38 +25,27 @@ resource "google_compute_instance_template" "default" {
   machine_type   = "e2-medium"
   can_ip_forward = false
 
-  tags = ["tag1", "tag2"]
-
   disk {
     source_image = data.google_compute_image.default.id
```
Comment (msampathkumar) [inline compute/zonal_instance_group_manager/main.tf:29]: Test it!

You don't have to define `data.google_compute_image`. You can use as following.

```suggestion
    source_image = "debian-cloud/debian-11"
```

Example: https://registry.terraform.io/providers/hashicorp/google-beta/latest/docs/resources/compute_instance_from_template

Comment (askubis) [reply compute/zonal_instance_group_manager/main.tf]: Thank you for that suggestion. I have done that (and removed the google_compute_image resource).

**Review by gericdong — APPROVED** [review]
LGTM

**Review by msampathkumar — APPROVED** [review]
LGTM.

**Pull request merged**